terraform/modules/bosh_vpc/variables.tf

```diff
@@ -67,6 +67,8 @@ variable "s3_gateway_policy_accounts" {
 }
 
 #Placeholder for real value, passed as a secret
-variable "block_range_20" {
-  default = "192.168.0.0/32"
-}
\ No newline at end of file
+variable "cidr_blocks" {
+  type    = list(string)
+  default = ["192.168.0.0/32", "192.168.0.1/32"]
```

Comment:
I don't think we want to set any default values. Because if we happened to not pass the real values through in CI automatically, then traffic from these default CIDRs would get blocked